src/Shopsys/ShopBundle/Resources/styles/front/common/components/box/filter.less

```diff
@@ -1,22 +1,47 @@
 @box-filter-padding: 10px;
 
 .box-filter {
+    margin-top: @margin-gap;
     margin-bottom: @margin-gap;
     padding: @box-filter-padding;
 
-    border: 1px solid @color-border;
+    .wrap-border;
+
+    @media @query-lg {
+        padding: 15px;
+    }
 
     &__price {
         margin-bottom: @margin-gap;
+
+        .form-line {
+            dt {
+                display: inline-block;
+                float: none;
+                width: 50px;
+            }
+
+            dd {
+                display: inline-block;
+                margin-left: 0;
+
+                .input {
+                    width: 80px;
+                }
+            }
+        }
+
+        .ui-slider {
```

Comment:
I don't understand ... why is this problem? I want indent ui-slider just in this case / place ... please show me excatly what is better solution.

Reply:
Slider could be wrapped, or can have class like this ".box-filter__slider" just in case js component will change.. so thats not a problem, just thinking in way - component has only element classes and even if you change component inside, you will not need to change ".ui-slider" class here